ledger/ledger-api-integration-tests/src/test/lib/scala/com/digitalasset/platform/apitesting/LedgerContextExtensions.scala

```diff
@@ -164,10 +164,12 @@ object LedgerContextExtensions extends Matchers with OptionValues {
         filters: Filters = Filters.defaultInstance)(
         implicit mat: ActorMaterializer,
         ec: ExecutionContext): Future[Assertion] = {
-      testingHelpers.submitAndVerifyFilterCantSeeResultOf(
-        createCommand(commandId, template, args, submitter),
-        TransactionFilter(Map(listener -> filters))
-      )
+      withClue("Creation of a template instance should not be visible to the listener") {
```

Comment:
Might be good to include information about what exactly happens: which template and submitter, what commandId etc.

Reply:
Done